tasks/post_copy_request_to_queue/test/unit_tests/test_post_copy_request_to_queue.py

```diff
@@ -63,39 +64,51 @@ def tearDown(self):
             "RETRY_BACKOFF": "2",
             "DATABASE_PORT": "5432",
             "DATABASE_NAME": "disaster_recovery",
-            "DATABASE_USER": "rhassan",
+            "DATABASE_USER": "orcauser",
         },
         clear=True,
     )
-    @patch("database.single_query")
-    @patch("requests_db.get_dbconnect_info")
+    @patch("post_copy_request_to_queue.shared_db.get_user_connection")
+    @patch("post_copy_request_to_queue.shared_db.get_configuration")
     @patch("post_copy_request_to_queue.shared_recovery.post_entry_to_queue")
     @patch("post_copy_request_to_queue.shared_recovery.update_status_for_file")
+    @patch("post_copy_request_to_queue.get_metadata_sql")  # todo: Unit test this sql?
```

Comment:
We probably want a basic happy-path test for this function now that SQLAlchemy makes checking the raw string harder.